src/graph_compute/chunk_scheduling.h

```diff
@@ -28,6 +28,8 @@ static inline bool interChunkDependency(vid_t v, vid_t w) {
   static const vid_t chunkMask = (1 << CHUNK_BITS) - 1;
   if ((v >> CHUNK_BITS) == (w >> CHUNK_BITS)) {
     return false;
+  } else if ((v & chunkMask) == (w & chunkMask)) {
```

Comment:
No, ((v & chunkMask) == (w & chunkMask)) implies that they are at the same position within their respective chunks (recall: the previous if which failed, checked if they are in the same chunk).